rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -743,75 +743,93 @@ bool main_window::InstallPackages(QStringList file_paths, bool from_boot)
 		m_gui_settings->SetValue(gui::fd_install_pkg, file_info.path());
 	}
 
-	if (file_paths.count() == 1 && file_paths.front().endsWith(".pkg", Qt::CaseInsensitive))
+	if (file_paths.count() == 1)
 	{
 		const QString file_path = file_paths.front();
 		const QFileInfo file_info(file_path);
 
-		compat::package_info info = game_compatibility::GetPkgInfo(file_path, m_game_list_frame ? m_game_list_frame->GetGameCompatibility() : nullptr);
-
-		if (!info.is_valid)
+		// #14398 - [Feature request] Install multiple PKG from command line
```

Comment:
This comment is unnecessary and adds nothing to the code.

Reply:
Removed